nat-lab/tests/test_core_api.py

```diff
@@ -49,6 +51,8 @@ class CoreApiErrorCode(Enum):
     "weight": int,
 }
 
+bearer_authorization_header = "Bearer IronKey"
```

Comment:
Afaik it should be more like `Bearer token:IronKey`

Reply:
I think that's the case for basic authentication.
In case of bearer token the header should be as following:
`"Authorization: Bearer <token>"`

Reply:
You're right. It's `Bearer token:<token>`